models/Group.ts

```diff
@@ -61,7 +68,7 @@ const stationLocationHasChanged = (
 
 const checkThatStationsAreNotTooCloseTogether = (
```

Comment:
Should we pull all the stations stuff out into it's own file?   It doesn't feel like it really belongs in Groups.

Reply:
Yeah, I think next time I make changes I might do that - probably can just live at the top of Stations model, since I don't think we really want to create "utility function" files for each model at this stage.